.github/workflows/release.yml

```
@@ -8,20 +8,18 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
     - name: Set up Python
-      uses: actions/setup-python@v4
+      uses: actions/setup-python@v5
       with:
-        python-version: 3.11
+        python-version: 3.12
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip build
-        pip install twine wheel
+        python -m pip install --upgrade pip build twine
```

I don't believe twine is needed here -- shouldn't the gh-action-pypi-publish action handle it if needed? [The docs](https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/) don't include its installation and `python -m build` works without it.

```suggestion
        python -m pip install --upgrade pip build
```